src/pages/settings/add-ons/rabbit-mq/RabbitMQ.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { useHistory } from "react-router-dom"
+import { useSelector } from "react-redux";
+import { Form, Col, Card, Input, Switch, Button } from "antd";
+import Sidenav from '../../../../components/sidenav/Sidenav'
+import Topbar from '../../../../components/topbar/Topbar'
+import ProjectPageLayout, { Content, InnerTopBar } from "../../../../components/project-page-layout/ProjectPageLayout";
+import { projectModules } from "../../../../constants";
+import FormItemLabel from "../../../../components/form-item-label/FormItemLabel"
+import { decrementPendingRequests, incrementPendingRequests, notify } from "../../../../utils";
+import { saveAddonConfig } from "../../../../operations/addons";
+
+const RabbitMQ = () => {
+    const history = useHistory()
+    const [form] = Form.useForm();
+    const config = useSelector(state => state.addonsConfig.rabbitmq);
+
+    const initialValues = config && config.enabled && {
+        cpu : config.resources.cpu,
+        memory: config.resources.memory,
+        highAvailability: config.options.highAvailability
+    }
+    
+    const handleFinish = (values) => {
+        const config = {
+            enabled: true,
+            resources: {
+                cpu: Number(values.cpu),
```

Comment:
cpu should be multiplied by 1000 while submitting here and divided by 1000 while binding to initialValues. Same for the other add on as well